pythonwhois/net.py

```diff
@@ -29,7 +29,7 @@ def get_whois_raw(domain, server="", previous=None, rfc3490=True, never_cut=Fals
 				is_exception = True
 				target_server = exc_serv
 				break
-		if is_exception == False:
+		if not is_exception:
```

Comment:
to make it even more Pythonic, is_exception can be removed completely and replaced by the for...else construct (see https://docs.python.org/2/tutorial/controlflow.html#break-and-continue-statements-and-else-clauses-on-loops)